### ISSUE TITLE
Tail stops working after some time

### ISSUE THREAD

**Issue by dennusb** [fluent/fluentd#1527]
- FluentD 0.12.31
- Kubernetes + Docker image version 1.23
- We have a few containers running on our Kubernetes cluster that have a mount point to the local storage on the Kubernetes worker host. They are logging data to that folder. Then the Fluentd container is also able to read the files in that specific folder and is performing a tail on those files.
As specified in the config : 

```
<source>
  type tail
  format /^(?<time>[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{3}) (?<Log-Level>[A-Z]+) (?<message>.*)/
  time_format %Y-%m-%d %H:%M:%S.%L
  path /var/log/node/**/*.tomcat.log
  pos_file /var/log/node/es-tomcat-simple.log.pos
  #read_from_head true
  tag tomcat.*
</source>
```

- The setup works well and files are tailed and imported in ElasticSearch by Fluentd. But for no apparent reason, mostly within 24 hours after starting the container the Fluentd process stops importing new lines from the tailed files. If we open a tail ourselves on that file we can see loads and loads of data are written there.
We also confirmed the td-agent is still running and that is has the file opened (checked with lsof).
If we delete the containers and spawn new ones, Fluentd immediately starts working again and new lines appear in Elastic Search.
The td-agent log is not showing any activity nor errors, so we are clueless about why this is happening and what is going wrong. Also we noticed that the pos files are no longer updated when this strange behaviour starts, but when we spawn new containers it continues where it stopped (according to the pos file).
Seems to be a bug with tailing files or something.

**Comment by repeatedly** [comment]
You can check which call or function is problem via sigdump or pid2line.rb.
Try it for more detail: http://docs.fluentd.org/v0.12/articles/trouble-shooting

**Comment by wangzhuzhen** [comment]
@dennusb 
@repeatedly 
I had met this before, and did  some debug after guiding by @repeatedly  and other nice guys. 
This may be something wrong between inotify and in_tail (via Cool.io), libev does not work properly, If so, try this:
in your in_tail configuration, set enable_watch_timer as true.

**Comment by dennusb** [comment]
Thanks for the input! We are now trying if this is fixing the issue on our acceptation environment. I will report back within 48 hours to let you know if it fixed it.

**Comment by dennusb** [comment]
It didn't fix the issue. After ±3 hours of processing it stopped again... any other ideas of what might fix this issue?

**Comment by dennusb** [comment]
I think it might be the same issue as described in this bug report : https://github.com/fluent/fluentd/issues/1383

**Comment by repeatedly** [comment]
Closed via #1383
